cruise-control-core/src/main/java/com/linkedin/cruisecontrol/monitor/sampling/aggregator/MetricValues.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2017 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.cruisecontrol.monitor.sampling.aggregator;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.Serializable;
+import java.nio.charset.StandardCharsets;
+import java.util.StringJoiner;
+
+
+/**
+ * A class hosting the values of a particular metric.
+ */
+public class MetricValues implements Serializable {
+  private static final long serialVersionUID = -6840253566423285966L;
```

Comment:
I've been meaning to remove all the serializable stuff. The internal format of the classes changes too often and so it's not useful.  So if you are not using it then you can just remove this.